### PR TITLE
Chore/adjust popover

### PR DESCRIPTION
**Pull request by alexandrejuk**
## Contexto

Este Pr refatora o componente header da pilot para user a nova ux/ui 

## Como testar?
Este pr adiciona uma nova prop para o component de popover

Como testar?

- Baixar o former-kit-skin-pagarme que possue a branch com o mesmo nome dessa
- Executar o yarn
- Executar o yarn build no former-kit-skin-pagarme
- Executar o yarn link no former-kit-skin-pagarme
- Baixar a branch chore/adjust-popover
- Executar o yarn
- executar o comando yarn link former-kit-skin-pagarme
- Executar o comando yarn install --force
- Executar o yarn link former-kit na pilot
e checar o component

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c1778f1d23c397097323aebfc1823e3424d8fc98
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c1778f1d23c397097323aebfc1823e3424d8fc98
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c1778f1d23c397097323aebfc1823e3424d8fc98
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c1778f1d23c397097323aebfc1823e3424d8fc98
  * lways use the standard `scope: action description`, sempre em inglês  
    - The **scope** shouldn't have spaces. If the word is composite, use dashes.  
    - The **action** should be **add**, **remove**, **refactor**, **move**,  **bump**, **update** or **release**  
    - The **description** should contain a quick description of the change

The complete commit style guide is available at https://github.com/pagarme/git-style-guide

---
This message was auto-generated by https://gitcop.com

**Review by zignd — APPROVED** [review]

File: packages/pilot/src/containers/Header/index.js [hunk 1]
```diff
@@ -88,19 +105,26 @@ const HeaderContainer = ({
...
+  const geralItems = [
+    { action: onBackToOldVersion, title: t('header.back_to_old_version') },
+    { action: onLogout, title: t('header.exit') },
+  ]
+
+  const geral = (
+    isPaymentLink(companyType)
+      ? geralItems.splice(0, 1)
+      : geralItems
+  )
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/index.js:124]: Ale, q q vc acha?

```suggestion
  const geralItems = [
    { action: onLogout, title: t('header.exit') }
  ]

  if (isPaymentLink(companyType)) {
    geralItems.unshift({ action: onBackToOldVersion, title: t('header.back_to_old_version') })
  }

File: packages/pilot/src/containers/Header/style.css [hunk 2]
```diff
@@ -29,3 +31,86 @@
...
+  width: 259px;
+}
+
+.popoverAccountInfo {
+  padding: var(--spacing-small);
+
+  & h4 {
+    color: var(--color-barney-purple-100);
+    font-weight: bold;
+    line-height: 24px;
```
Comment (eduardogspereira) [inline packages/pilot/src/containers/Header/style.css:46]: Ale, nesses espacamentos da para usar os espacos do former-kit-skin-pagarme. A galera também prefere que os tamanhos sempre sejam multiplos de 8.., entao ali no 259px precisaria ser 264px. Vlw!

**Comment by eduardogspereira** [comment]
Ficou bem fera ale! :rocket:

**Review by eduardogspereira — APPROVED** [review]

**Review by zignd — APPROVED** [review]

**Pull request merged**